tests/ui/tox.ini

```diff
@@ -0,0 +1,11 @@
+# content of: tox.ini , put in same dir as setup.py
+[tox]
+envlist = ui-tests
+[testenv]
+basepython = python2.7
+install_command = pip install --no-deps {packages}
+[testenv:ui-tests]
+passenv = DISPLAY
```

Comment:
Remove this for now, it might come back when we get the tests running in Travis CI or CircleCI though.